.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+  - name: ESLint
+    url: https://github.com/eslint/eslint/issues
+    about: Report problems with the ESLint program itself to the ESLint project.
```

Feel free to suggest changes to `name`, `url` and `about` text. 😉

---

Not really sure what's the idea with these links.

---

The idea is that you'll have a button on this screen: https://github.com/testing-library/eslint-plugin-testing-library/issues/new/choose with a link to the ESLint repository.

For an example of how those buttons look go here: https://github.com/progit/progit2/issues/new/choose

---

Aaah ok. I think this one is fine then. Should we add a link to our Discussions then?

- name: Ask a Question, Discuss
- URL: https://github.com/testing-library/eslint-plugin-testing-library/discussions
- about: Look for `eslint-plugin-testing-library` Questions and Discussions or start a new one

---

```suggestion
  - name: Ask a Question, Discuss
    url: https://github.com/testing-library/eslint-plugin-testing-library/discussions
    about: Look for `eslint-plugin-testing-library` Questions and Discussions or start a new one
```

I'm not sure if we can use `monospaced text` in the `about` section... That part might not work properly. If things break we can just use normal plain text in the field. 😄

---

Oh sorry, I didn't mean to replace the ESLint one but having both!

---

Whoops! 😄 I'll go add the ESLint one back as well. 😉